tests/functional/pod_and_daemons/test_mds_cache_trim_standby.py

```diff
@@ -0,0 +1,104 @@
+    def teardown(self, request):
+        def finalizer():
+            """
+            This function will call a function to clear the mds memory usage gradually
+
+            """
+            cluster.bring_down_mds_memory_usage_gradually()
+
+        request.addfinalizer(finalizer)
+
+    @pytest.mark.polarion_id("OCS-6280")
+    def test_mds_cache_trim_on_standby_replay(self, dc_pod_factory):
+        """
+        Verifies whether the MDS cache is trimmed or not in standby-replay mode.
+
+        Steps:
+        1. Run metadata IO on at least 5 pods, so that the memory consumption will be rapid and high.
+        2. When mds memory utilisation reached 75% [it is similar to 150% of cache utilisation], there is a possibility
+        to trigger MDS cache oversized warning.
+        3. Now, the script look for such warnings in ceph health.
+        4. If found the warning, it identifies whether the warning is from Active or standby mds
+         by searching the warning in the mds pod logs.
+        5. After that, it identifies whether the cache trim is happening or not on stand by MDS.
+        6. Test case passes if MDS cache oversized warning not found in Standby mds and
+          if cache trim is happening in Standby mds pod.
+
+        """
+        run_metadata_io_with_cephfs(dc_pod_factory, no_of_io_pods=5)
+        log.info(
+            "Starting metadata IO in the background. Monitoring for MDS cache alerts."
+        )
+
+        trim_msgs = ["cache trim"]
+        cache_warning = "MDSs report oversized cache"
+
+        for sampler in TimeoutSampler(
+            timeout=1800,
+            sleep=20,
+            func=cluster.get_active_mds_memory_utilisation_in_percentage,
+        ):
+            if sampler > 75:
+                break
+            else:
+                log.warning("MDS memory consumption is not yet reached target")
+
+        active_mds_mem_util = cluster.get_active_mds_memory_utilisation_in_percentage()
+        sr_mds_mem_util = (
+            cluster.get_standby_replay_mds_memory_utilisation_in_percentage()
+        )
+
+        log.info(f"Active MDS memory utilization: {active_mds_mem_util}%")
+        log.info(f"Standby-replay MDS memory utilization: {sr_mds_mem_util}%")
+        ceph_health_detail = cluster.ceph_health_detail()
+
+        standby_replay_mds_log = get_pod_logs(
+            pod_name=cluster.get_mds_standby_replay_info()["standby_replay_pod"]
+        )
+
+        cache_trim_validation = [
+            msg for msg in trim_msgs if msg in standby_replay_mds_log
+        ]
+
+        assert (
+            cache_trim_validation
+        ), f"Cache trim messages not found in standby-replay MDS logs: {standby_replay_mds_log}"
+
+        log.info("MDS cache trim is happening on standby-replay MDS")
+
+        if cache_warning not in ceph_health_detail:
+            log.info("No cache oversized warnings detected in Ceph health details.")
+        elif cache_warning in standby_replay_mds_log:
```

Comment:
if there is an oversized cache warning it standby mds, that won't appear in ceph health detail output?